### ISSUE TITLE
:sparkles: Support Variadic Positional Arguments

### ISSUE THREAD

**Issue by tombulled** [neoclient/neoclient#29]
For example, support:
```python
@get("/{path_params}")
def foo(*path_params: str) -> None:
    ...
```
and:

```python
@get("/")
def foo(**query_params: str) -> None:
    ...
```

**Comment by tombulled** [comment]
Related to #22

**Comment by tombulled** [comment]
TODO: Merge this issue with #22

**Comment by tombulled** [comment]
Closed as duplicate of #22
